app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/settings/components/OptionScaffold.kt

```diff
@@ -42,7 +44,11 @@ fun OptionScaffold(
     navigateBack: (() -> Unit)? = null,
 ) {
     Scaffold(topBar = {
-        val canDisplayBackButton = navigateBack?.let { TopAppBarButton.backButton(navigateBack) }
+        val canDisplayBackButton = if (LocalNavType.current == NavigationSuiteType.NavigationBar) {
```

Comment:
Why this TopAppBarButton is named like a boolean ?

Reply:
Yeah, this makes no sens, I renamed it